tests/conformance/state/state.go

```diff
@@ -474,4 +474,72 @@ func ConformanceTests(t *testing.T, props map[string]string, statestore state.St
 		features := statestore.Features()
 		assert.False(t, state.FeatureETag.IsPresent(features))
 	}
+
+	if config.HasOperation("first-write") {
```

Comment:
This is great! Thanks. BTW, can you add a scenario with first-write and an etag to update an item? Something like:

+ Write an item
+ Read item and get eTag (eTag1)
+ Set with FIRST_WRITE and eTag as eTag1
+ Read item again and check that eTag (eTag2 != eTag1) and value changes
+ Set again with FIRST_WRITE and eTag as eTag1
+ Check the set above failed.

Reply:
Done.